src/Foundation/Config.php

```diff
@@ -42,7 +40,7 @@ public function inMaintenanceMode(): bool
         return $this->data['offline'] ?? false;
     }
 
-    private function requireKeys(...$keys)
+    public function requireKeys(...$keys)
```

Comment:
Not sure how I feel about mutating the singleton... I would prefer that we pass in required keys to the constructor, or make this method return a clone of the config object

Reply:
Agreed, will update and fix tests in one swoop.

Reply:
This won't work. The construct of this class is fired from the site.php which is too easily accessible from the outside. Either we drop outside mutation of the required keys from outside (demanding them based on a flag perhaps $installed as argument) or we use the above change.

Reply:
In this case, let's temporarily remove requiring _any_ keys, and revisit validation later? The previous config object didn't have external validation, so adding it now isn't necessary (plus, the choice of 'url' as the main attribute to enforce needs more discussion in retrospect)

Reply:
Patched tests, removed method.